packages/botonic-core/src/models/legacy-types.ts

```diff
@@ -316,3 +316,9 @@ export type MatchingProp =
   | 'request'
 
 export type Matcher = string | RegExp | ((args) => boolean)
+
+export enum BotonicAction {
+  Redirect = 'redirect_action:',
+  CreateCase = 'create_case:',
+  CreateTestCase = 'create_test_integration_case:',
+}
```

Comment:
i would change this a little bit in favour of something like:
```

type BotonicActionSeparator = ':';

enum BotonicActionType {
  Redirect = 'redirect_action',
  CreateCase = 'create_case',
  CreateTestCase = 'create_test_integration_case',
}

type BotonicAction = `${BotonicActionType}${BotonicActionSeparator}${string}`;

```

To be more precise, here an example: 
const action:BotonicAction = `${BotonicActionType.Redirect}:blablabla`
And this will make tsc to fail: 
const action:BotonicAction = `${BotonicActionType.Redirect}--blablabla`

Also in this way we can have this value in the Session correctly typed